documentation/modules/post/windows/gather/packrat_credentials.md

```diff
@@ -0,0 +1,139 @@
+## Module Description
+
+This post-exploitation module gathers artifacts from end users systems.
+
+Artifacts include 13 Browers, 12 IM (chat) applications, 6 Email clients and 1 Game
+These artiacts and then scraped for credentials (usernames/passwords) and custom regular expressions
+
+6 Email clients:
+Incredimail, Outlook, Opera Mail, PostBox Mail, Mozilla Thunderbird Mail, Windows Live Mail
+
+12 IM (chat):
+AIM (Aol Instant Messaging), Digsby, GaduGadu, ICQ, Miranda, Nimbuzz, Pidgen, QQ (Chinese), Skype, Tango, Tlen.pl (Polish), Trillian, Viber, xChat
+
+1 Game:
+Xfire
+
+13 Browser:
+Avant, Comodo, CoolNovo, Chrome, FireFox, Flock, IE, K-Meleon, Maxthon, Opera, SRware, Safari, SeaMonkey
+
+## Verification Steps
+
+1. Start MSF console
+2. Get a Meterpreter session on a Windows system
+3. use post/windows/gather/credentials/packrat_credentials
+4. Set SESSION 1
+5. enter 'run' to extract credentials from all applications
+
+
+
+## Options 'APPLICATIONS'
+
+Users can enter APPLICATIONS to extract from example output shown below for email service incredimail
+
+msf post(windows/gather/credentials/packrat_credentials) > set SESSION 1
```

Comment:
You'll want to put code in [code blocks](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code).